Cargo.toml

```diff
@@ -12,4 +12,5 @@ opt-level = 2
 
 [patch.crates-io]
 der = { git = "https://github.com/RustCrypto/formats.git" }
```

Comment:
@baloo please drop these two patches. We should not have such overrides after a relase